generative_data_prep/utils/path_verify.py

```diff
@@ -18,11 +18,14 @@
 Check to make sure that the files can be created and correctly written or read from.
 If the path is not valid, then fail out gracefully with ValueError.
 """
+import logging
 import os
 import shutil
 from typing import Optional
 
-from .utils import SEP_STR
+from .logger import log_sep_str
+
+logger = logging.getLogger("generative_data_prep_logger")
```

Comment:
is there a reason we don't want to use `__name__` here? That would include module hierarchy in the logs and make it easier to debug.

```
logger = logging.getLogger(__name__)

Reply:
I did not create a module hierarchy logger, this logger is not for debugging, but for the user to understand what kind of data was prepared, ie what flags they passed in, and metrics associated with it.